catalog-rest-service/src/main/java/org/openmetadata/catalog/security/DefaultAuthorizer.java

```diff
@@ -59,8 +59,6 @@ public void init(AuthorizerConfiguration config, Jdbi dbi) throws IOException {
     this.userRepository = new UserRepository(collectionDAO);
     mayBeAddAdminUsers();
     mayBeAddBotUsers();
-    policyEvaluator = PolicyEvaluator.getInstance();
-    policyEvaluator.setPolicyRepository(new PolicyRepository(collectionDAO));
```

Comment:
This does not get called in NoopAuthorizer and hence can lead to NPE for PolicyEvaluator while refreshing Rules